src/components/panels/details/info-multiple-selection.tsx

```diff
@@ -51,66 +33,87 @@ export default function InfoMultipleSection() {
+        <>
+            <Box>
+                <Heading as="h5" size="sm">
+                    {t('panel.details.multipleSelection.selected')} {selected.size}
+                </Heading>
+                <VStack m="var(--chakra-space-1)">
+                    <HStack w="100%">
+                        <Heading as="h5" size="xs" w="100%">
+                            {t('Show')}
+                        </Heading>
+                        <RmgButtonGroup
+                            selections={[
+                                {
+                                    label: t('panel.details.multipleSelection.station'),
+                                    value: 'station',
+                                },
+                                {
+                                    label: t('panel.details.multipleSelection.miscNode'),
+                                    value: 'misc-node',
+                                },
+                                {
+                                    label: t('panel.details.multipleSelection.edge'),
+                                    value: 'line',
+                                },
+                            ]}
+                            defaultValue={['station', 'misc-node', 'line']}
+                            multiSelect={true}
+                            onChange={value => setFilter(value as FilterType[])}
+                        />
+                    </HStack>
+                    {filter.length !== 0 && (
+                        <>
+                            <Button width="100%" size="sm" onClick={() => setIsOpenChangeModal(true)}>
+                                {t('panel.details.multipleSelection.change')}
+                            </Button>
+                            <Divider />
+                        </>
+                    )}
+                    {[...selected]
+                        .filter(id => filter.includes('station') || !id.startsWith('stn'))
+                        .filter(id => filter.includes('misc-node') || !id.startsWith('misc'))
+                        .filter(id => filter.includes('line') || !id.startsWith('line'))
+                        .map(id => (
+                            <HStack key={id} width="100%">
+                                <Button
+                                    width="100%"
+                                    size="sm"
+                                    variant="solid"
+                                    onClick={() => dispatch(setSelected(new Set([id])))}
+                                    overflow="hidden"
+                                    maxW="270"
+                                    textOverflow="ellipsis"
+                                    whiteSpace="nowrap"
+                                    display="ruby"
+                                >
+                                    {getName(id)?.replaceAll('\\', '⏎')}
+                                </Button>
+                                <Checkbox
```

Comment:
Either we use checkbox but do not delete the item when it is unchecked, or we put two button here one as unselecting the element and another one as deleting the element. Current behavior will remove the selection permentantly when it is unchecked :(
Also, when filter, elements could be left with unchecked status but still displayed.

Reply:
Is there any possible we temporarily unchecked some of selected items to do some actions? Why not just unselect? 👀

Reply:
Shall we remove a selected one by clicking the button with shift key?

Reply:
It's ok to not temporarily uncheck some of the selected items. But in this case, a button with __X__ will be confusing as it may also indicate a permanent removal. Another option is to have two buttons one with __X__ and another one with _grabarge_ icon for permanent removal.